helm/charts/hpe-csi-driver/values.yaml

```diff
@@ -18,3 +18,5 @@ disableNodeConformance: false
 iscsi:
   chapUser: ""
   chapPassword: ""
+
+registryPrefix: "quay.io/"
```

Comment:
Can we move "/" to the yaml instead of user specifying it explicitly?
registryPrefix: "quay.io"

Reply:
I thought we should be able to remove the prefix by giving an empty string and use the image as hpestorage/csi-driver:tag or our internal registry.

Reply:
The default registryPrefix should be "quay.io/hpestorage". We want to give customers control which repo they want to use for our images. If registryPrefix is an empty string, we can fall back to "hpestorage", and then images will be pulled from system default registry and our preferred default repository. This automatically removes the trailing "/" problem too because the initial slash will always be present in the template before the image name.

Reply:
> The default registryPrefix should be "quay.io/hpestorage". We want to give customers control which repo they want to use for our images. If registryPrefix is an empty string, we can fall back to "hpestorage", and then images will be pulled from system default registry and our preferred default repository. This automatically removes the trailing "/" problem too because the initial slash will always be present in the template before the image name.

Ok, this breaks CSI sidecars. New plan, just get rid of trailing slash in registryPrefix and rename the parameter to "registry". If "registry" is set to empty string, don't append the slash to make up the valid fully qualified image name. This will allow system default registries to be pulled from.

Reply:
@datamattsson  does the change look fine now?